scripts/shared/clusters.sh

```diff
@@ -295,7 +295,9 @@ function deploy_cluster_capabilities() {
 
 function warn_inotify() {
     if [[ "$(cat /proc/sys/fs/inotify/max_user_instances)" -lt 512 ]]; then
-        echo "Please increase your inotify settings (currently $(cat /proc/sys/fs/inotify/max_user_watches) and $(cat /proc/sys/fs/inotify/max_user_instances)):"
+        echo "Your inotify settings are lower than our recommendation"
+        echo "This may cause failures in large deployments, but we don't know if it caused this failure"
```

Comment:
```suggestion
        echo "Your inotify settings are lower than our recommendation."
        echo "This may cause failures in large deployments, but we don't know if it caused this failure."
```